[Performance feature] Error deploying feature on E2E use case

This is the job executed:
```
if [ -f "features/performance/Makefile" ];then
    cd features/performance
	ISOLATED_CPUS=2 RESERVED_CPUS=2 make
else
    exit 1
fi
```

This is the Jenkins Output:

```
Started by upstream project "TE-UC01" build number 36
originally caused by:
 Started by user admin
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/E2E-Steps/Worker-Performance
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] Done
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/openshift-kni/baremetal-deploy.git
 > git init /var/lib/jenkins/workspace/E2E-Steps/Worker-Performance # timeout=10
Fetching upstream changes from https://github.com/openshift-kni/baremetal-deploy.git
 > git --version # timeout=10
 > git fetch --tags --progress -- https://github.com/openshift-kni/baremetal-deploy.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/openshift-kni/baremetal-deploy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/openshift-kni/baremetal-deploy.git # timeout=10
Fetching upstream changes from https://github.com/openshift-kni/baremetal-deploy.git
 > git fetch --tags --progress -- https://github.com/openshift-kni/baremetal-deploy.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 6ceaead20d5a777505315e137d08b2ede4772a0b (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 6ceaead20d5a777505315e137d08b2ede4772a0b # timeout=10
Commit message: "Provide scripts to generate and deploy performance manifests (#36)"
 > git rev-list --no-walk 6ceaead20d5a777505315e137d08b2ede4772a0b # timeout=10
[EnvInject] - Executing scripts and injecting environment variables after the SCM step.
[EnvInject] - Injecting as environment variables the properties content 
KUBECONFIG=/var/lib/jenkins/.kube/e2e_kubeconfig

[EnvInject] - Variables injected successfully.
[Worker-Performance] $ /bin/sh -xe /tmp/jenkins212272282312616800.sh
+ '[' -f features/performance/Makefile ']'
+ cd features/performance
+ ISOLATED_CPUS=2
+ RESERVED_CPUS=2
+ make
./hack/generate.sh
./hack/deploy.sh
node/worker-0 labeled
machineconfigpool.machineconfiguration.openshift.io/master patched
machineconfigpool.machineconfiguration.openshift.io/worker patched
error: must specify one of -f and -k
make: *** [Makefile:3: deploy] Error 1
Build step 'Execute shell' changed build result to UNSTABLE
Finished: UNSTABLE
```

--- a/features/performance/hack/deploy.sh
+++ b/features/performance/hack/deploy.sh
@@ -17,7 +17,7 @@ for mcp in ${mcps}; do
 done
 
 # apply manifests
-oc apply -R ${MANIFESTS_GENERATED_DIR}
+oc apply -Rf ${MANIFESTS_GENERATED_DIR}
 
 # unpause all machine config pools
 for mcp in ${mcps}; do

